Sources/StreamVideoSwiftUI/Utils/ClipCorners.swift

```diff
@@ -0,0 +1,101 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+/// `CornerClipper` is a ViewModifier that clips a SwiftUI view
+/// to have rounded corners on specified sides.
+struct CornerClipper: ViewModifier {
```

Comment:
not related to the PR: we have similar thing for the chat SDK, we might want to introduce view utils package at some point.